.gitignore

```diff
@@ -1,3 +1,4 @@
 _site/
 .sass-cache/
 .jekyll-metadata
+*~
```

Comment:
Ideally this change would be in a separate PR - It's clearly valuable, but probably doesn't need to be tied up with the issues that may arise here.